content/rule_book/material_and_questions.md

```diff
@@ -19,215 +42,234 @@
-|-----------------|--------:|--------:|----------------------|
-| Interrogative   |     8/9 |   14/16 | INT                  |
-| Multiple Answer |       1 |       2 | MA                   |
-| Reference       |       3 |     6/7 | CR, CVR, MACR, MACVR |
-| Quote           |     2/3 |     3/4 | Q, Q2V               |
-| Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
-| Situation       |     2/0 |     4/0 | SIT                  |
+The only exception to this is Situation type question quotations, which could potentially involve a single quotation, broken up by non-quotation words. This would result in the Situation quotation being written without the non-quotation words in the middle of the 2 quotations.
 
-**Question Type Minimums**
-: For a given Question Type, the minimum number of questions must be asked in Question Numbers 1-20. This excludes any alphanumeric question numbers.
+On some question types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
 
-**Question Type Maximums**
-: For a given Question Type, the maximum number of questions must not be exceeded in all "non-Overtime" questions. This includes any alphanumeric question numbers.
+### Question Types
 
-**Overtime Specifics**
-: Question Type maximums can be exceeded in Overtime.
+#### Interrogative (INT)
 
-## Question Writing
+Interrogative questions, also called "standard" questions or "regular" questions or "normal" questions:
 
-### Preparation of Questions
+- Must have exactly 1 inserted interrogative word at the very beginning of the question or the very end of the question
+- Must have a single answer
+- Must include a unique word or a 2 or 3-word unique phrase within the 1st 5 words of the question
 
-- All questions must contain verbatim material.
-    - The only exception to this is Situation question Quotations, which could potentially involve a single Quotation, broken up by non-quotation words. This would result in the Situation Quotation being written without the non-quotation words in the middle of the two quotations.
-- On some Question Types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+#### Multiple Answer (MA)
 
-### Question Types
+- Must meet all the requirements of an INT except for the single answer requirement
+- Must have more than 1 answer
+    - The clarification of a single answer does not meet the above requirement
+    - The identification of a single pronoun does not meet the above requirement
+    - The identification of a plural pronoun does meet the above requirement
 
-#### Interrogative (INT)
+In addition, an MA question is invalid if 1 of the listed answers to the multiple answer's questions is a "negative" answer. A "negative" answer is an answer used to contrast another answer or an answer that does not answer the question, such as an answer that begins with the words or similar words to "not" or "neither".
```

Comment:
Do we want "> [remainder of question text]"  under INT/MA to be uber consistent/explicit?

Reply:
I don't understand the question. Can you elaborate a bit or provide an example?

Reply:
You put what the QM needs to say for Q/Ref, but not for INT/MA/F. Even though what the QM needs to say for the reading of INT/MA/F is much simpler than Q/Ref, do you want to state it to be explicit?

Reply:
I don't think it's necessary, but we could revisit in a v2.